libs/core/execution_base/include/hpx/execution_base/completion_signatures.hpp

```diff
@@ -25,6 +25,8 @@
 #include <system_error>
 #include <type_traits>
 #include <utility>
+#include "hpx/config/forward.hpp"
+#include "hpx/functional/tag_invoke.hpp"
```

Comment:
Please always #include HPX headers using the `<...>` syntax. Also, please group all HPX headers in one place (like above).